### PR TITLE
make external, rm empty funcs, add events

### PR DESCRIPTION
**Pull request by RensR**
Static analysis results

### CODE REVIEW

File: contracts/src/v0.8/ccip/offRamp/EVM2EVMMultiOffRamp.sol [hunk 1]
```diff
@@ -84,6 +84,7 @@ contract EVM2EVMMultiOffRamp is IAny2EVMMultiOffRamp, ITypeAndVersion, MultiOCR3
   /// @dev RMN depends on this event, if changing, please notify the RMN maintainers.
   event CommitReportAccepted(CommitReport report);
   event RootRemoved(bytes32 root);
+  event LatestPriceEpochAndRoundSet(uint40 oldEpochAndRound, uint40 newEpochAndRound);
```
Comment (elatoskinas) [inline contracts/src/v0.8/ccip/offRamp/EVM2EVMMultiOffRamp.sol:87]: q: actually wondering if we need to set this with OCR3? Is there any use-case for the reset of the sequenceNumber?

The LiquidityManager's OCR3Base has this check in `transmit`:

```solidity
    if (sequenceNumber <= s_latestSequenceNumber) {
      revert NonIncreasingSequenceNumber(sequenceNumber, s_latestSequenceNumber);
    }
```

Which looks like this should always increase

**Review by elatoskinas — APPROVED** [review]
LGTM

File: contracts/src/v0.8/ccip/offRamp/EVM2EVMMultiOffRamp.sol (hunk 1 above)
Comment (RensR) [reply contracts/src/v0.8/ccip/offRamp/EVM2EVMMultiOffRamp.sol:87]: Might be a good question for research if this makes sense. Nice cleanup if we can remove it

Comment (RayXpub) [reply contracts/src/v0.8/ccip/offRamp/EVM2EVMMultiOffRamp.sol:87]: Do we need to emit both old and new values?

Comment (RensR) [reply contracts/src/v0.8/ccip/offRamp/EVM2EVMMultiOffRamp.sol:87]: I'd say that it is useful yes. This is not hot-path so gas doesn't matter and it clearly tells the story of what happened through the logs this way

**Review by RayXpub — APPROVED** [review]

**Pull request merged**